skbio/io/format/fasta.py

```diff
@@ -84,9 +84,10 @@
 ~~~~~~~~~~~~~~~
 Each sequence header consists of a single line beginning with a greater-than
 (``>``) symbol. Immediately following this is a sequence identifier (ID) and
-description separated by one or more whitespace characters. Both sequence ID
-and description are optional and are represented as the empty string (``''``)
-in scikit-bio's objects if they are not present in the header.
+description separated by one or more whitespace characters. The sequence ID and
+description are stored in the sequence metadata attribute. Both are optional.
```

Comment:
can you mark `metadata` in backticks? since it's an attribute you're referencing